app/js/HealthChecker.js

```diff
@@ -60,8 +63,14 @@ module.exports = {
           }
         })
       },
-      (cb) => db.docs.remove({ _id: doc_id, project_id }, cb),
-      (cb) => db.docOps.remove({ doc_id }, cb)
+      (cb) =>
+        docsCollectionPromise.then((docs) =>
+          docs.deleteOne({ _id: doc_id, project_id }, cb)
+        ),
+      (cb) =>
+        docOpsCollectionPromise.then((docOps) =>
+          docOps.deletedOne({ doc_id }, cb)
```

Comment:
```suggestion
          docOps.deleteOne({ doc_id }, cb)
```
typo, sorry for missing that.

Reply:
Thanks, I'll fix it once https://github.com/overleaf/chat/pull/56 has landed and I can port the change over.

Reply:
Done https://github.com/overleaf/docstore/compare/4e8bc097c0bf6ed2ce6f9f3472e8bf0c6fea6e07..dcd5bf1d6bfaff7f8f47bcfe339c7ade9601d858#diff-faf70888ab0205343a737ebd2358ec5c